README.rst

```diff
@@ -140,7 +140,23 @@ Skipping Tests
 ~~~~~~~~~~~~~~
 
 Doctest provides the ``+SKIP`` directive for skipping statements that should
-not be executed when testing documentation. However, it is often useful to be
+not be executed when testing documentation.
+
+.. code-block:: python
+
+    >>> open('file.txt') # doctest: +SKIP
+
+And in `.rst` documentation, whole code example blocks can be skipped with the
```

Comment:
Please use double backticks. It won't matter here much as we don't send the README through sphinx, but still good measure.